packages/wrangler/src/deploy/deploy.ts

```diff
@@ -760,6 +764,7 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 					}
 				}
 			} catch (err) {
+				printBindings({ ...withoutStaticAssets, vars: maskedVars });
```

Comment:
Having this in the catch... could it result in printing the bindings twice? What happens if an error is thrown after the bindings have already been printed?

Reply:
It seemed to me like the code after `printBindings` isn't fallible as it's just a few prints, I guess that could change in the future.
I could add a flag to check if the bindings have already been printed if you feel like it's more appropriate.

Reply:
Added a flag.